VikasFashionsAPI/APIServices/CompanyFiscalYearService/ICompanyFiscalYearService.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace VikasFashionsAPI.APIServices.CompanyFiscalYearService
+{
+    public interface ICompanyFiscalYearService
+    {
+        Task<CompanyControl?> GetCompanyControlByUserIdAsync(int userId);
```

Comment:
CompanyControl  method should be in company control service instead of here (CompanyFiscalYearService).